backend/src/v5/models/modelSettings.js

```diff
@@ -72,4 +72,37 @@ Models.getFederations = async (ts, ids, projection, sort) => {
 	return findModel(ts, query, projection, sort);
 };
 
+Models.updateModelSettings = async (ts, model, payload) => {
```

Comment:
should probably call it something else, like `data`

payload is a bit too specific to network requests.